DOCKERHUB-COMPANION.md

```diff
@@ -0,0 +1,21 @@
+# Kroki Companion Image
+
+Image to use together with this server: [yuzutech/kroki](https://hub.docker.com/r/yuzutech/kroki)
+
+GitHub repository: [kroki](https://github.com/yuzutech/kroki)
```

Comment:
I think it would be better to link to the user documentation no?

https://docs.kroki.io/kroki/setup/install/#_images

```suggestion
Read the [Kroki installation](https://docs.kroki.io/kroki/setup/install/#_images) to learn how to start Kroki companion containers.
```

Reply:
Having said that it would be useful to add the GitHub repository but I don't know where to configure it:

![image](https://user-images.githubusercontent.com/333276/118892910-ea2f3400-b901-11eb-84a3-8331c81fe034.png)


![image](https://user-images.githubusercontent.com/333276/118892961-fc10d700-b901-11eb-9a64-4b3d393e352e.png)

😞

Reply:
That's what I suspected, thanks for the investigation.

>  when you use automated builds in Dockerhub that's not an option I presume.

Indeed, not an option for us :(